core/tests/webdriverio_desktop/explorationStatisticsTab.js

```diff
@@ -124,7 +124,6 @@ describe('Statistics tab', function() {
     await explorationPlayerPage.submitAnswer('TextInput', '2');
     await explorationPlayerPage.viewHint();
     await explorationPlayerPage.submitAnswer('TextInput', '3');
-    await explorationPlayerPage.viewSolution();
```

Comment:
Why don't we test viewing the solution anymore? That seems like an important thing to check since that functionality still exists based on your videos

Reply:
The project involves removing the dependence of solution of hints both on the learner and the editor side. Earlier, the solution was to be released just like a hint, which was being basically tested here. Since that dependence is removed, removed the testing for this.

Reply:
OK, approving since we test viewing the solution separately here: 

https://github.com/oppia/oppia/blob/6d6e7bac8ad211cba2cdde23f3a0258f10dd1c64/core/tests/webdriverio_desktop/additionalPlayerFeatures.js#L307